src/main/resources/db/changelog/db.changelog-master.yaml

```
@@ -0,0 +1,5 @@
+databaseChangeLog:
+  - include:
+      file: db/changelog/changes/01-create-book-table.yaml
+  - include:
+      file: db/changelog/changes/02-insert-books.yaml
```

```suggestion
      file: db/changelog/changes/02-insert-books.yaml

```
forgot an empty line